packages/tslint-config/src/rules/blueprintClassesConstantsRule.ts

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
hmm, this package is only intended for `@blueprintjs` packages, right? So it doesn't make sense to put rules that will be consumed by end-user apps here, does it?

Reply:
🤔 after this PR, it will contain some blueprint-specific rules, but this package is intended for consumption by who-so-ever wishes. 

i proposed putting these rules here or in `core` itself and we agreed that this package makes more sense as it actually depends on `tslint`. i'll migrate the internal tslint-config to depend on this (where most of the config came from in the first place).

Reply:
https://github.com/palantir/tslint-blueprint was designed for rules like this, consider putting these there

Reply:
Seems a little late for that! Looks like that package already has this rule. But makes sense for these things to live in the blueprint repo, no?

Also 👋👋 hi @adidahiya!!